src/main/js/config/FormControls.tsx

```diff
@@ -209,24 +209,37 @@ export const createPeriodOptions: (t: (key: string) => string) => SelectItem[] =
   {
     label: t("scm-repository-mirror-plugin.form.period.options.oneDay"),
     value: "1440"
+  },
+  {
+    label: t("scm-repository-mirror-plugin.form.period.options.disabled"),
+    value: "0"
   }
 ];
 
 export const SynchronizationPeriodControl: FC<MirrorAccessConfigControlProps> = ({ control, isReadonly }) => {
   const [t] = useTranslation("plugins");
   const { field } = useController({ control, name: "synchronizationPeriod", defaultValue: 60 });
+  const disabled = !field.value;
 
   return (
-    <Column className="column is-full">
-      <Select
-        label={t("scm-repository-mirror-plugin.form.period.label")}
-        helpText={t("scm-repository-mirror-plugin.form.period.helpText")}
-        options={createPeriodOptions(t)}
-        disabled={isReadonly}
-        testId="synchronization-period-input"
-        {...field}
-      />
-    </Column>
+    <>
+      <Column className="column is-half">
+        <Select
+          label={t("scm-repository-mirror-plugin.form.period.label")}
+          helpText={t("scm-repository-mirror-plugin.form.period.helpText")}
+          options={createPeriodOptions(t)}
+          disabled={isReadonly || disabled}
+          testId="synchronization-period-input"
+          {...field}
+        />
+        <Checkbox
+          label={t("scm-repository-mirror-plugin.form.syncDisabled.label")}
+          helpText={t("scm-repository-mirror-plugin.form.syncDisabled.helpText")}
+          onChange={d => field.onChange(d? null: 60)}
+          checked={disabled}
+        />
```

Comment:
Do we really want both? A checkbox to disable the mirroring and also an option in the select component? Seems kinda awkward.

Reply:
Ups :-(